src/ssl.c

```diff
@@ -25980,12 +26078,98 @@ WOLFSSL_X509 *wolfSSL_d2i_X509_fp(XFILE fp, WOLFSSL_X509 **x509)
 
 
 #ifdef HAVE_CRL
+
+#ifndef NO_BIO
+WOLFSSL_API WOLFSSL_X509_CRL *wolfSSL_d2i_X509_CRL_bio(WOLFSSL_BIO *bp, 
+                                                    WOLFSSL_X509_CRL **x)
+{
+    int derSz;
+    byte* der = NULL;
+    WOLFSSL_X509_CRL* crl = NULL;
+    
+    if ((derSz = wolfSSL_BIO_get_len(bp)) > 0) {
```

Comment:
Please sanitize "bp" for NULL before using.  It doesn't look like wolfSSL_BIO_get_len() adequately sanitizes.

Reply:
Agree.